.github/workflows/build.yml

```diff
@@ -66,17 +66,31 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: ./scripts/ci-select-xcode.sh 16.3
+      - run: brew install xcodegen
+        if: ${{ contains(matrix.scheme, 'SessionReplay-CameraTest') }}
+      - run: make session-replay-camera-test-xcode
+        if: ${{ contains(matrix.scheme, 'SessionReplay-CameraTest') }}
 
       # Note: Due to complexity in implementing the CODE_SIGNING_ALLOWED flag in the sentry-xcodebuild.sh script,
       #       we did not yet migrate this step to use the script yet.
-      - run: >-
+      - if: ${{ !contains(matrix.scheme, 'SessionReplay-CameraTest') }}
+        run: >-
           env NSUnbufferedIO=YES
           xcodebuild
           -workspace Sentry.xcworkspace
           -scheme '${{matrix.scheme}}'
           -configuration Debug
           CODE_SIGNING_ALLOWED="NO"
           build
+      - if: ${{ contains(matrix.scheme, 'SessionReplay-CameraTest') }}
+        run: >-
+          env NSUnbufferedIO=YES
+          xcodebuild
+          -project Samples/${{matrix.scheme}}/${{matrix.scheme}}.xcodeproj
+          -scheme '${{matrix.scheme}}'
+          -configuration Debug
+          CODE_SIGNING_ALLOWED="NO"
+          build
```

Comment:
`l`: Does it make sense to migrate this to `sentry-xcodebuild.sh` in this PR?

Reply:
no, i'd prefer not to touch that script here, and based on https://github.com/getsentry/sentry-cocoa/pull/5106#discussion_r2055063454, might even change this back.